recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "python-suitesparse-graphblas" %}
-{% set version = "8.2.1.0" %}
+{% set version = "9.3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/suitesparse-graphblas/suitesparse-graphblas-{{ version }}.tar.gz
-  sha256: 69c975d9b5ddcf5fa3172a86c3ccbdac4670667a39bac41138b2d70fe24e32ba
+  url: https://files.pythonhosted.org/packages/source/s/suitesparse-graphblas/suitesparse_graphblas-{{ version }}.tar.gz
```

We have started recommending `pypi.org` as the preferred location

```suggestion
  url: https://pypi.org/packages/source/s/suitesparse-graphblas//suitesparse_graphblas-{{ version }}.tar.gz
```

---

ok, thanks, we were having "not found" issues these urls. I found one that works (changed a hyphen to an underscore):
```
https://pypi.org/packages/source/s/suitesparse_graphblas/suitesparse_graphblas-9.3.1.0.tar.gz
```

---

Fixed, thanks.